### PR TITLE
Feature: UI support for DP Error Notifications

### PR DESCRIPTION
**Pull request by Guaris**
https://konghq.atlassian.net/browse/DOCU-3645

### CODE REVIEW

**Review by acgoldsmith — APPROVED** [review]
test

File: app/konnect/updates.md [hunk 1]
```diff
@@ -11,6 +11,11 @@ services. [Try it today!](https://cloud.konghq.com/quick-start)
 
 ## March 2024
 
+**Enhanced error logging in {{site.konnect_short_name}} UI**
+: Customers now have the ability to view detailed error logs for their data plane directly within {{site.konnect_short_name}}. The enhanced logging feature categorizes errors for improved clarity, distinguishing between configuration errors, transient configuration reload errors, and general exceptions. This update empowers users to diagnose and resolve CP/DP connection issues more efficiently.
```
Comment (hl782) [inline app/konnect/updates.md:15]: error logs --> error notifications

**Review by hl782 — APPROVED** [review]
just one small change but approved otherwise

File: app/konnect/updates.md (hunk 1 above)
Comment (Guaris) [inline app/konnect/updates.md:15]:
```suggestion
: Customers now have the ability to view detailed error notifications for their data plane directly within {{site.konnect_short_name}}. The enhanced logging feature categorizes errors for improved clarity, distinguishing between configuration errors, transient configuration reload errors, and general exceptions. This update empowers users to diagnose and resolve CP/DP connection issues more efficiently.
```

File: app/konnect/updates.md [hunk 2]
```diff
@@ -11,6 +11,11 @@ services. [Try it today!](https://cloud.konghq.com/quick-start)
 
 ## March 2024
 
+**Enhanced error logging in {{site.konnect_short_name}} UI**
```
Comment (Guaris) [inline app/konnect/updates.md:14]:
```suggestion
**Enhanced error notification in {{site.konnect_short_name}}**
```

**Pull request merged**